record/record.app.js

```diff
@@ -50,6 +52,8 @@
                     $rootScope.relatedReferences = $rootScope.reference.related;
                     // There should only ever be one entity related to this reference
                     return $rootScope.reference.read(1);
+                }, function error(exception) {
+                    return $q.reject(exception);
```

Comment:
I tested with invalid schema/table/column name, where resolve rejects the promise, but that wasn't being caught. This fixed that problem.